include/aws/crt/Api.h

```diff
@@ -125,12 +125,46 @@ namespace Aws
             /// @private
             static const Io::IsTlsAlpnSupportedCallback &GetBYOCryptoIsTlsAlpnSupportedCallback();
 
+            /**
```

Comment:
It's not your changes, but the includes on this header file are way too much.  Maybe see what can be removed (with appropriate forward declarations).